src/DssSpell.t.sol

```diff
@@ -302,18 +286,15 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(lerp.done());
     }
 
-    function testNewChainlogValues() public { // make private to disable
+    function testNewChainlogValues() private { // make private to disable
```

Comment:
Nit: the version is already tested via the base test, but if you updated this test, maybe you should make it public